src/main/recon/httpx.ts

```diff
@@ -17,12 +17,15 @@ export async function liveSubDomains(
   )} -o ${path.join(outputDir, 'httpx_live_domains.txt')}`;
   try {
     const res = await execAsync(command);
-    const domainsFound = resultFromStd(res.stderr, /\bFound (\d+) subdomains?/);
+    const domainsFound = resultFromStdHTTPX(
+      res.stderr,
+      /^(https?:\/\/)?([a-zA-Z0-9.-]+)\.([a-zA-Z]{2,})(\/[^\s]*)?$/i,
+    );
     const db = connectJson(path.join(`${outputDir}/details.json`));
     await db.update({
       recon: {
         liveDomains: {
-          result: parseInt(domainsFound, 10),
+          result: domainsFound,
```

Comment:
This return an array, so it should be array.length()

Reply:
I actually do it look to teh fun imp